sign-pkg-mac/action.yaml

```diff
@@ -111,8 +105,10 @@ runs:
         done
 
         # don't forget the application itself
-        cp -a ".app/$app_name.app" "$volpath/"
-        echo "app_path=$volpath/$app_name.app" >> "$GITHUB_ENV"
+        artifact_app="$(ls -dt .app/*.app | head -n 1)"
```

Comment:
is there any useful prefix to use for defensive sanity checking?  possibly something like `.app/Second*.app` or somesuch?

Reply:
The only way the app gets there is from the Mac build step. As you saw, the previous logic tried to predict the app_name based on replicating part of the logic from viewer_manifest.py. Instead of more completely replicating the logic, signing up for maintenance in two places, I think it's best to leave viewer_manifest.py solely responsible for the app name.